library/Pdfexport/ProvidedHook/Pdfexport.php

```diff
@@ -66,7 +66,25 @@ public function htmlToPdf($html)
         // the object is destructed
         $chrome = $this->Chrome();
 
-        return $chrome->fromHtml($html)->toPdf();
+	$pdf = $chrome->fromHtml($html)->toPdf();
+
+	if ($html instanceof PrintableHtmlDocument && ($coverPage = $html->getCoverPage()) !== null) {
+            $coverPagePdf = $chrome
+                ->fromHtml((new PrintableHtmlDocument())
+                    ->add($coverPage)
+                    ->addAttributes($html->getAttributes())
+                    ->removeMargins()
+                )
+                ->toPdf();
+
+            $merger = new Merger(new TcpdiDriver());
+            $merger->addRaw($coverPagePdf);
+            $merger->addRaw($pdf);
+
+            $pdf = $merger->merge();
+	}
+
+	return $pdf;
```

Comment:
Oops, didn't saw that previously. The entire block is not indented correctly. We use 4 spaces everywhere.

Reply:
fixed indention to 4 spaces